src/Spotify/SpotifyBackend.h

```diff
@@ -10,12 +10,15 @@
 #define _SPOTIFYBACKEND_H_
 
 #include "MusicBackend.h"
+#include "SpotifyAPI.h"
+#include "SpotifyAuthorization.h"
 #include "Types/GlobalTypes.h"
 #include "Types/Queue.h"
 #include "Types/Result.h"
 
 class SpotifyBackend : public MusicBackend {
  public:
+  virtual ~SpotifyBackend();
```

Comment:
if you just want to stop the server, why not do it in the destructor of `SpotifyAuthorization`?

Reply:
haha yes you're right, will change that 😺